packages/splunkProcessor/src/splunkProcessor.js

```diff
@@ -104,25 +104,51 @@ function transformLambdaLogEvent(logEvent) {
 function transformLogEvent(logEvent, logGroup, accountNumber) {
```

Comment:
I wouldn't say this is very readable. How about splitting it into individual functions for lambda, stepfunction and other logs?
The whitespace is a bit inconsistent, too.